torch/_dynamo/guards.py

```diff
@@ -1222,6 +1222,13 @@ def add_code_part(code_part, guard, log_only=False):
         # guard_fn.__globals__ becomes equal to builder.scope. This causes
         # guard_fn to hold a referece to f_locals sitting in builder.scope["L"]
         globals_for_guard_fn = {"G": builder.scope["G"]}
+        from torch.utils._triton import has_triton
+
+        if has_triton() and "triton" in pycode:
```

Comment:
```suggestion
        if has_triton():
```
This will make it not break @anijain2305's C++ guards

Reply:
ah we didn't need the "triton" in pycode, that was left over from my debugging, we can unconditionally add triton to the global scope